components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -7222,7 +7223,8 @@ def getTiles(self, zctTileList):
                 exc_info=True)
             exc = e
         try:
-            rawPixelsStore.close()
+            if rawPixelsStore is not None:
+                rawPixelsStore.close()
```

Comment:
So, this simply handles the case that ```_prepareRawPixelsStore()``` returns ```None``` or throws an exception?

Reply:
it handles the case when an exception is thrown during setPixelsId (done in ``_prepareRawPixelsStore()``)